db/seeds.rb

```diff
@@ -20,11 +17,21 @@
 frosty = User.create!(name: 'Frosty', email: 'snow@brr.com', birthdate: Date.new(2001,2,3))
 santa = User.create!(name: 'Nick', email: 'slay@northpole.com', birthdate: Date.new(1950,12,25))
 
-FactoryBot.create_list(:era, 3, user: frosty)
-FactoryBot.create_list(:era, 3, user: santa)
+Era.create(name: 'childhood', start_date: Date.new(2001,2,3), end_date: Date.new(2011,2,3), start_week: Random.rand(0..300), end_week: Random.rand(301..500), color: Faker::Color.hex_color, user: frosty)
```

Comment:
Re-seeded, added question_type. 

Ran locally and seeded. Worked through this on a call with FE